Lambda-School-Labs/CS10-quizzer
# Abbott/server urls fix

# Description

Adds various React URL's to Django's urls.py file, fixes path resolution failures when attempting to visit a path (such as `/rocket/billing`) directly, or refreshing a page.

## Type of change


# How Has This Been Tested?

- Navigate to the client directory and run `yarn build`
- Once the client is built, launch the Django server in the root directory
- Visit `127.0.0.1:8000` in your browser, create a new account or sign into an existing one (though with the latest changes from Brandon you may need to delete and rebuild the database)
- Navigate to any page using the rocketlist, and then refresh that page, it will no longer provide an error screen.
    - Alternatively, directly navigate to a path such as `/rocket/billing/`, you will be greeted by the appropriate page instead of an error message.

# Checklist:
